docs/sources/features/explore/index.md

```diff
@@ -51,6 +51,45 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 You can close the newly created query by clicking on the Close Split button.
 
+## Query history
```

Comment:
I think Query history should be in a separate topic.

Reply:
But Rich history is only available in Explore. Couldn't it be confusing for users?

Reply:
The current Explore topic is currently too complex and needs to be broken up. It's a little bit nuts.

I guess it won't do any more harm to keep it in there. I'll deal with it when I get around to rewriting the Explore documentation.